install_config/configuring_openstack.adoc

```diff
@@ -157,3 +157,38 @@ When triggering a containerized installation, only the directories of
 container. Therefore, *_cloud.conf_* should be in *_/etc/origin/_* instead of
 *_/etc/_*.
 ====
+
+[[openstack-applying-configuration-changes]]
+== Applying Configuration Changes
+
+Start or restart {product-title} services on all master and node hosts to apply your
+configuration changes:
+
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+# systemctl restart atomic-openshift-node
+----
+endif::[]
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master
+# systemctl restart origin-node
+----
+endif::[]
+
+Switching from not using a cloud provider to using a cloud provider produces an
+error message. Adding the cloud provider tries to delete the node because the
+node switches from using the *hostname* as the `*externalID*` (which would have
+been the case when no cloud provider was being used) to using the OpenStack
+`*instance-id*` (which is what the OpenStack cloud provider specifies). To resolve
+this issue:
+
+.  Log in to the CLI as a cluster administrator.
```

Comment:
How about adding a step  to backup the node labels.

Reply:
@qinpingli Sorry I do not know those steps. Can you provide me with the instructions to backup all labels and restoring them?

Reply:
Just because step 4 told the cluster admin to add back node labels, so we should remind the cluster admin to check the node labels before deleting node.
$ oc get node <node-name> -o yaml cmd can get the node label.

Reply:
@qinpingli I used `$ oc describe node <node_name> | grep -Poz '(?s)Labels.*\n.*(?=Taints)'` command to just display the labels.

Reply:
LGTM